### PR TITLE
Adding README and cdevents-visulaiser project under poc

### PR DESCRIPTION
**Pull request by rjalander**
Adding README content for reference implementation details, before we start.
This PoC is as per the presentation made on the Workgroup meetings, https://github.com/cdevents/presentations/pull/25

### CODE REVIEW

**Review by adamkenihan — APPROVED** [review]

File: poc/visualisation_reference_architecture.jpg
Comment (elmsdata) [inline poc/visualisation_reference_architecture.jpg:1]: Great diagram! Are we going to support multiple types of databases and have separate connectors for each one?

**Comment by afrittoli** [comment]
Is there anything preventing this from merging (apart from the DOC check)?

File: poc/visualisation_reference_architecture.jpg
Comment (rjalander) [reply poc/visualisation_reference_architecture.jpg:1]: Yes, In this PoC we used ArangoDB as one of the implementations to process the CDEvent, similar way many other implementations with different types of databases can be implemented to receive the CDEvent and process.

**Comment by afrittoli** [comment]
@rjalander it would be nice to merge this PR - if you have a chance could you squash the commits and fix the DCO check, so we can merge it?

**Comment by rjalander** [comment]
@afrittoli fixed DCO check and squashed commits, can we merge this poc project.
I will create another PR to include the visualization with grafana changes once this PR is merged.

**Review by afrittoli — APPROVED** [review]

**Comment by afrittoli** [comment]
@rjalander - the commit is not signed, but I'll bypass this time :)

**Pull request merged**